package.json

```diff
@@ -2332,12 +2332,11 @@
         "remove-files-webpack-plugin": "^1.4.0",
         "rewiremock": "^3.13.0",
         "rimraf": "^3.0.2",
-        "screenshot-capture": "^0.2.1",
         "screenshot-desktop": "^1.12.7",
         "sinon": "^8.0.1",
         "source-map": "^0.7.3",
         "source-map-support": "^0.5.12",
-        "string_decoder": "^1.3.0",
+        "string_decoder": "~1.1.1",
```

Comment:
1.3.0 was never used anywhere. & this was throwing errors when running `npm list --production --parseable --depth=99999 --loglevel=error` which is run by `vsce` when building the vsix package